### PR TITLE
LXQt: use sddm as default configured display manager

### PR DESCRIPTION
**Pull request by DimStar77**
- Related ticket: https://progress.opensuse.org/issues/115385
- Needles: 
- Verification run: https://openqa.opensuse.org/tests/2538863#live

### CODE REVIEW

**Comment by DimStar77** [comment]
This was changed in TW with https://build.opensuse.org/request/show/995113

The Leap tests don't get that far anyway at the moment and I suspect strongly to get this also into Leap 15.5, which is why I did not add special handling for versions (i.e. all versions tested are the same)

**Comment by DimStar77** [comment]
Actually, Leap 15.5 already caught up to this: https://build.opensuse.org/request/show/996454

File: tests/x11/lxqt_reconfigure_openqa.pm [hunk 1]
```diff
@@ -18,9 +18,9 @@ sub run {
 
     set_var("DESKTOP", "lxqt");
 
-    # LXQt uses lightdm as window manager, which has the user preselected
-    set_var('DISPLAYMANAGER', 'lightdm');
-    # LightDM has the user in a drop-down preselected; there is no need to type the username
+    # LXQt uses sddm as window manager, which has the user preselected
+    set_var('DISPLAYMANAGER', 'sddm');
+    # sddm asks straoght for PW with only one user; there is no need to type the username
```
Comment (Vogtinator) [inline tests/x11/lxqt_reconfigure_openqa.pm:23]:
```suggestion
    # sddm asks straight for PW with only one user; there is no need to type the username
```

**Review by Vogtinator — APPROVED** [review]
Leap maintenance doesn't test LXQt either, so that should be fine.

+1 once the typo is fixed.

**Pull request merged**